react/alert/Alert.tsx

```diff
@@ -0,0 +1,48 @@
+import classNames from 'classnames';
+import { AlertProps } from './props.ts';
+import { ExpandTransition } from '../expandtransition/index.ts';
+import { WIcon } from '../icon/index.ts';
+import style from 'inline:./style.css';
+
+const iconMap = {
```

Comment:
Maybe iconMap & iconClass could be an enum of objects?

const iconEnum = {
negative: {
    name: 'Error',
    class: 's-icon-negative',
},
.... };